pages/boac/boac_cohort_pages.rb

```diff
@@ -202,9 +202,14 @@ def sort_by_entering_term
     sort_by 'Entering Term'
   end
 
-  # Sorts cohort search results by units
-  def sort_by_units
-    sort_by 'Units Completed'
+  # Sorts cohort search results by units completed
```

Comment:
@pfarestveit Comment typo?

Reply:
I have completed fixing the typo @paulkerschen !